diff --git a/NavHeader.tsx b/NavHeader.tsx
index 736580bc3d350d932f427cd8dbb8da7855909a64..f845c76180615d59f145e20b90df0be18e08df55 100644
--- a/NavHeader.tsx
+++ b/NavHeader.tsx
@@ -162,7 +162,7 @@ const renderMobileItem = (
           <path
             d="M2.21289 12H0.291016L5.73047 0H7.65234L2.21289 12Z"
             fill="white"
-            fill-opacity="0.5"
+            fillOpacity={0.5}
           />
         </svg>
       </span>
